tests/test_telemetry.py

```diff
@@ -359,3 +359,47 @@ def test_app_dependencies_loaded(self):
         for dependency, seen in seen_loaded_dependencies.items():
             if not seen:
                 raise Exception(dependency + " not recieved in app-dependencies-loaded message")
+
+    def setup_app_product_change(self):
+        weblog.get("/enable_product")
+
+    @bug(
```

Comment:
When an endpoint is missing, we categorize it as missing feature.

```suggestion
    @missing_feature(context.library in ("dotnet", "nodejs", ...),   reason="weblog GET/enable_product and app-product-change event is not implemented yet.")
```